src/com/facebook/buck/jvm/common/BUCK

```diff
@@ -0,0 +1,13 @@
+java_library(
+  name = 'util',
```

Comment:
Hmm, @shs96c is not going to be happy about having a `util` library.  What about putting this under a resources package instead?